### PR TITLE
Release Ready

### PR DESCRIPTION
**Pull request by sgoggins**
**Description**
- Completed Testing

This PR fixes #
- issue #2787 
- issue #2786 
- issue #2785 
- issue #2752 
- issue #2656 
- issue #2542 
- issue #2439 
- issue #2289 

**Signed commits**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sgoggins** [comment]
There are some typos, mostly related to `absoulte` as a spelling for `absolute`, but lets just let it ride since the error is a variable name and consistent throughout. At least for this release.

**Review by Ulincsys — CHANGES_REQUESTED** [review]
Double-checking the linter output aired some issues, and also a few spelling mistakes that should be fixed.

File: augur/application/db/lib.py [hunk 1]
```diff
@@ -95,3 +105,408 @@ def get_value(section_name: str, setting_name: str) -> Optional[Any]:
...
+        return [row.access_token for row in results]
+    
+def get_active_repo_count(collection_type):
+
+    with get_session() as session:
+    
+        return session.query(CollectionStatus).filter(getattr(CollectionStatus,f"{collection_type}_status" ) == CollectionState.COLLECTING.value).count()
+
+
+def facade_bulk_insert_commits(logger, records):
```
[the remark this answers is not in the document]
Comment (sgoggins) [reply augur/application/db/lib.py:200]: @ABrain7710

File: augur/tasks/github/util/github_task_session.py [hunk 2]
```diff
@@ -35,10 +37,28 @@ class GithubTaskSession(DatabaseSession):
...
+    #def __init__(self, logger: Logger, engine=None):
     def __init__(self, logger: Logger, engine=None):
 
         super().__init__(logger, engine=engine)
-
-        self.oauths = GithubRandomKeyAuth(self, logger)
-        self.platform_id = 1
         
+        self.oauths = GithubRandomKeyAuth(logger)
+        #self.oauths = GithubRandomKeyAuth(self, logger) # Removed and replaced for the issue below in frontend.py
```
Comment (IsaacMilarky) [inline augur/tasks/github/util/github_task_session.py:46]: ?

**Review by Ulincsys — APPROVED** [review]

**Review by IsaacMilarky — APPROVED** [review]
Looks good for the most part.

**Pull request merged**